Packs/PrismaCloud/Integrations/RedLock/RedLock.yml

```diff
@@ -250,6 +308,21 @@ script:
     arguments:
     - name: alert-id
       description: comma separated list of string IDs to be dismissed
+    - name: snooze-value
+      required: false
+      description: The amount of time to snooze. Both snooze value and unit must be specified
+    - name: snooze-unit
+      required: false
+      auto: PREDEFINED
+      predefined:
+      - hour
+      - day
+      - week
+      - month
+      - year
+      - login
```

Comment:
delete
```suggestion
```

Reply:
*Login is removed as snooze option
*We also confirmed that the remediate command does work as expected, it just now takes about 20 min on the PCS side *to remove the alert after it has been remediated.  
*Removed remediate step from test playbook

Reply:
working on the camel case piece still...

Reply:
A reminder: when you change the context outputs int the python file you should also change the outputs in the yml file correspondingly